core/src/SYCL/Kokkos_SYCL.cpp

```diff
@@ -224,10 +224,6 @@ std::ostream& SYCL::impl_sycl_info(std::ostream& os,
             << device.get_info<device::global_mem_cache_size>()
             << "\nGlobal Mem Size: "
             << device.get_info<device::global_mem_size>()
-            << "\nMax Constant Buffer Size: "
-            << device.get_info<device::max_constant_buffer_size>()
-            << "\nMax Constant Args: "
-            << device.get_info<device::max_constant_args>()
```

Comment:
Yes, this pull request originally contained all the changes to avoid using deprecated code wrt to the compiler version I was using to compose this pull request in (separate commits, see [`00b5531` (#4183)](https://github.com/kokkos/kokkos/pull/4183/commits/00b553153adf220ca8251e68b99b10cb9e66db06), [`234d6f2` (#4183)](https://github.com/kokkos/kokkos/pull/4183/commits/234d6f2f7912e8430efe04f44e3bde7d41d1d843), https://github.com/kokkos/kokkos/pull/4183/commits/2557434904629fd935b17b60a4a6fe222578f400). Most of them I needed to revert again, see [`2de95e2` (#4183)](https://github.com/kokkos/kokkos/pull/4183/commits/2de95e29ada9bdc2bf9647f4bd2183e7650a343f), [`fea18fd` (#4183)](https://github.com/kokkos/kokkos/pull/4183/commits/fea18fd530bc40335be64ac4bb5268030356c72c), since the version we are testing with doesn't support the new names yet.

I am currently preferring to perform these updates as drive-by so that I don't need to keep track of them all the time when working on multiple pull requests. Anyway, I can remove them if desired.

Reply:
Now that's fine by me